ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml

```diff
@@ -0,0 +1,196 @@
+base_images:
+  builder:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  cluster-logging-operator:
+    name: "6.1"
+    namespace: logging
+    tag: cluster-logging-operator
+  cluster-logging-operator-e2e:
+    name: "6.1"
+    namespace: logging
+    tag: cluster-logging-operator-e2e
+  cluster-logging-operator-registry:
+    name: "6.1"
+    namespace: logging
+    tag: cluster-logging-operator-registry
+  elasticsearch-operator:
+    name: "5.8"
+    namespace: logging
+    tag: elasticsearch-operator
+  elasticsearch-operator-registry:
+    name: "5.8"
+    namespace: logging
+    tag: elasticsearch-operator-registry
+  elasticsearch-proxy:
+    name: 5.y
+    namespace: logging
+    tag: elasticsearch-proxy
+  log-file-metric-exporter:
+    name: 6.y
+    namespace: logging
+    tag: log-file-metric-exporter
+  logging-elasticsearch6:
+    name: 5.y
+    namespace: logging
+    tag: logging-elasticsearch6
+  logging-eventrouter:
+    name: 5.y
+    namespace: logging
+    tag: logging-eventrouter
+  logging-fluentd:
+    name: v1.16.x
+    namespace: logging
+    tag: logging-fluentd
+  logging-kibana6:
+    name: 5.y
+    namespace: logging
+    tag: logging-kibana6
+  oauth-proxy:
+    name: "4.14"
+    namespace: ocp
+    tag: oauth-proxy
+  ubi9-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_path: Dockerfile
+  from: ubi9-minimal
+  to: vector
+- dockerfile_path: Dockerfile.unit
+  from: builder
+  to: logging-vector-test-unit
+promotion:
+  to:
+  - name: "6.1"
+    namespace: logging
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 20Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make test
+  container:
+    from: logging-vector-test-unit
+- as: cargo-deny-check-bans
+  commands: cargo deny --no-default-features --features ocp-logging --offline check
+    bans
+  container:
+    from: logging-vector-test-unit
+- as: ldd-openssl
+  commands: ldd /usr/bin/vector | grep -E 'libcrypto|libssl'
+  container:
+    from: vector
+- as: cluster-logging-operator-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: obs-logging
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.15"
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: export GINKGO_OPTS="-ginkgo.skip fluentd"; FAKE_HOME_DIR=/tmp/home
```

Comment:
@jcantrill can we remove: `GINKGO_OPTS="-ginkgo.skip fluentd";`?

Reply:
yes